src/accessors/java/org/spongepowered/common/accessor/world/level/block/entity/AbstractFurnaceBlockEntityAccessor.java

```diff
@@ -51,4 +57,9 @@
 
     @Accessor("recipeType") RecipeType<? extends AbstractCookingRecipe> accessor$recipeType();
 
+    @Invoker("canBurn")
+    static boolean invoker$canBurn(@Nullable Recipe<?> var0, NonNullList<ItemStack> var1, int var2) {
+        throw new UnsupportedOperationException("Untransformed shadow");
```

Comment:
There is an exception to throw for this, UntransformedInvokerException

Reply:
Blame copy paste from the shadow that existed before